css/cptui.scss

```diff
@@ -200,13 +200,20 @@ $max-tablet: "screen and (max-width: 768px)";
   	}
 	.cptui-about-text {
 		margin-bottom: 1em !important;
+		margin-right: 0px;
+		max-width: calc(100% - 173px);
 	}
-	.cptui-badge {
-		position: absolute;
-	    top: 0;
-	    right: 0;
+}
+.email-octopus-form-row {
+	input::placeholder {
+		color: #cccccc;
 	}
```

Comment:
Let's keep the cptui-badge selector as it puts the graphic in the upper right corner.

Reply:
Ah, I'm seeing some markup changes below related to this. If I recall right, I probably borrowed heavily from BuddyPress/bbPress for layout for this.

Still would be interesting to see about alignment options.